notebooks/information_extraction_raven.ipynb

```diff
@@ -0,0 +1,2726 @@
+{
```

Comment:
<p>Please replace the link with <a href="https://docs.haystack.deepset.ai/v2.0/docs/custom-components" target="_blank">https://docs.haystack.deepset.ai/docs/custom-components</a>, without /v2.0/ suffix. Also the links above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>